app/repositories/CommonBSONDocuments.scala

```diff
@@ -60,6 +60,37 @@ trait CommonBSONDocuments {
     )
   }
 
+  def inviteToTestBSON[P <: ProgressStatus](targetProgressStatus: P, invigilatedKeyToExclude: String) = {
```

Comment:
it's not entirely common. do we need a new common trait just for tests? Otherwise we will end up having something like: OnlineTestApplication (which sounds like common thing, but contains things connected with etray, video interview etc.)